CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Bump xamltools to 17.12.35312.88 (PR: [#7562](https://github.com/dotnet/vscode-csharp/pull/7562))
```

Comment:
this actually goes into the previous section.  we do our weekly snap on Friday at 4pm, until then we're still in 2.48